src/map/party.c

```diff
@@ -300,6 +300,18 @@ int party_recv_info(const struct party *sp, int char_id)
 			if (i == MAX_PARTY)
 				added[added_count++] = member_id;
 		}
+		
+		ARR_FIND(0, MAX_PARTY, j, p->party.member[j].leader == 1);
+		if (j == MAX_PARTY) {
+			// Leader has changed
+			int k;
+			ARR_FIND(0, MAX_PARTY, k, sp->member[k].leader == 1);
+			if (k < MAX_PARTY) {
+				clif->PartyLeaderChanged(map->id2sd(sp->member[k].account_id), 0, sp->member[k].account_id);
+			} else {
+				party->broken(p->party.party_id); // Should not happen, Party is leaderless, disband
```

Comment:
I know that this is just a sanity check and it should never happen, but you can't call this here. Everything will blow up below, when it tries to edit a party that was just deleted (and its memory released).